hack/release-notes.md

```diff
@@ -6,7 +6,7 @@ Find out on [our blog](https://blog.argoproj.io) and [changelog](https://github.
 
 ## Breaking Changes and Known Issues
 
-Can be found in the [installation guide](https://argoproj.github.io/argo-workflows/installation/).
+Check out [upgrading guide](https://github.com/argoproj/argo-workflows/blob/master/docs/upgrading.md) and search for [existing issues on GitHub](https://github.com/argoproj/argo-workflows/issues).
```

Comment:
```suggestion
Check out [upgrading guide](https://argoproj.github.io/argo-workflows/upgrading/) and search for [existing issues on GitHub](https://github.com/argoproj/argo-workflows/issues).
```

The docs normally link to the hosted docs instead of the repo. Was there a reason you used the repo link?